level: ERROR 
time: 2017-04-10 02:06:05,222 
path: /Users/davy/kadai-s-project/find_ubike_nearby/api/views.py 
message: 'dict' object has no attribute 'lat'

level: ERROR 
time: 2017-04-10 02:06:32,952 
path: /Users/davy/kadai-s-project/find_ubike_nearby/api/views.py 
message: 'dict' object has no attribute 'lat'

